contracts/src/LightClient.sol

```diff
@@ -121,6 +131,16 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         BN254.ScalarField blockCommRoot;
     }
 
+    /// @notice Simplified HotShot commitment struct
+    /// @param l1BlockHeight the block height of l1 when this state update was stored
+    /// @param l1BlockTimestamp the block timestamp of l1 when this state update was stored
+    /// @param hotShotCommitment The HotShot commitment info of the latest finalized HotShot block
+    struct StateHistoryCommitment {
+        uint256 l1BlockHeight;
```

Comment:
@alysiahuggins Have you looked at the gas usage of using uint256 vs for example u64 here? The values should both fit safely within a u64 and I think we might save a word or two of storage if we use u64.

Reply:
i mean long live ethereum lol but yes agreed, u65 is more practical , will address

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/785bf92c65397aca1b323275a5b058bae60810ca